File: src/main/java/fr/marstech/mtlinkspray/service/DashboardService.java
package fr.marstech.mtlinkspray.service;

import fr.marstech.mtlinkspray.dto.DashboardDto;

public interface DashboardService {

    DashboardDto createDashboard(DashboardDto dashboardDto);

    DashboardDto createDashboard(String dashboardName);

    DashboardDto getDashboard(String id);
}

File: src/main/kotlin/fr/marstech/mtlinkspray/service/DashboardService.kt
package fr.marstech.mtlinkspray.service

import fr.marstech.mtlinkspray.dto.DashboardDto

interface DashboardService {
    fun createDashboard(dashboardDto: DashboardDto): DashboardDto

    fun createDashboard(dashboardName: String): DashboardDto

    fun getDashboard(id: String): DashboardDto

    fun updateDashboard(id: String, dashboardDto: DashboardDto): DashboardDto
}
